proxy/private_rpc/pr_eth_transaction_api.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any, ClassVar
+
+from common.ethereum.errors import EthError, EthNonceTooLowError, EthNonceTooHighError, EthWrongChainIdError
+from common.http.utils import HttpRequestCtx
+from common.jsonrpc.errors import InvalidParamError
+from common.neon.account import NeonAccount
+from common.neon.transaction_model import NeonTxModel
+from common.utils.json_logger import logging_context
+from .pr_eth_signing_api import PrEthSigningApi
+from .server_abc import PrivateRpcApi, PrivateRpcServerAbc
+from ..base.mp_api import MpTxRespCode
+
+
+class PrEthTransactionApi(PrivateRpcApi):
+    name: ClassVar[str] = "PrivateRpc::EthTransaction"
+
+    def __init__(self, server: PrivateRpcServerAbc) -> None:
+        super().__init__(server)
+        self._signing_api = PrEthSigningApi(server)
+
+    @PrivateRpcApi.method(name="eth_sendTransaction")
+    async def eth_send_transaction(self, ctx: HttpRequestCtx, tx: Dict[str, Any]) -> str:
+        if "from" not in tx:
+            raise InvalidParamError(message="no sender in transaction")
+
+        if (neon_account := NeonAccount.from_raw(tx["from"], ctx.chain_id)) is None:
```

Comment:
Chain-id can be defined in the `tx`.
If it isn't defined, should be used the default-chain-id